python/test/model_TEST.py

```diff
@@ -19,11 +19,11 @@
 from gz.common import set_verbosity
 from gz_test_deps.sim import TestFixture, Model, World, world_entity
 
-post_iterations = 0
-iterations = 0
-pre_iterations = 0
-
 class TestModel(unittest.TestCase):
+    k_null_entity = 0
```

Comment:
I was hoping we would use the constant from https://github.com/gazebosim/gz-sim/blob/579013fe2841de68c0901192780ab8ee86cc9bb1/include/gz/sim/Entity.hh#L61. Sorry for not specifying clearly.

Reply:
Done.